module ca.ucalgary.groupprojectgui.p3 {
    requires javafx.controls;
    requires javafx.fxml;
    requires javafx.web;

    requires org.controlsfx.controls;
    requires com.dlsc.formsfx;
    requires net.synedra.validatorfx;
    requires org.kordamp.ikonli.javafx;
    requires org.kordamp.bootstrapfx.core;
    requires eu.hansolo.tilesfx;
    requires com.almasb.fxgl.all;

    opens ca.ucalgary.groupprojectgui.p3 to javafx.fxml;
    exports ca.ucalgary.groupprojectgui.p3;
}